### PR TITLE
[doc][yba] Backport node agent remove port ref

### PR DESCRIPTION
**Pull request by ddhodge**
Remove more references to port for node agent (see #22440)

@netlify /preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual/#reconfigure-a-node-agent

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md [hunk 1]
```diff
@@ -698,13 +698,13 @@ To reconfigure a node for use in a different provider, do the following:
...
     ```sh
-    node-agent node configure -t <api_token> -u https://<yba_address>:9000
+    node-agent node configure -t <api_token> -u https://<yba_address>
     ```
 
     For example, if you run the following:
 
     ```sh
-    node-agent node configure -t 1ba391bc-b522-4c18-813e-71a0e76b060a -u http://10.98.0.42:9000
+    node-agent node configure -t 1ba391bc-b522-4c18-813e-71a0e76b060a -u http://10.98.0.42
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md:707]:
```suggestion
    node-agent node configure -t 1ba391bc-b522-4c18-813e-71a0e76b060a -u https://10.98.0.42
```

File: docs/content/stable/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md [hunk 2]
```diff
@@ -616,13 +616,13 @@ To install the YugabyteDB node agent manually, as the `yugabyte` user, do the fo
...
    ```sh
-   ./installer.sh -c install -u https://<yba_address>:9000 -t <api_token>
+   ./installer.sh -c install -u https://<yba_address> -t <api_token>
    ```
 
    For example, if you run the following:
 
    ```sh
-   ./installer.sh  -c install -u http://10.98.0.42:9000 -t 301fc382-cf06-4a1b-b5ef-0c8c45273aef
+   ./installer.sh  -c install -u http://10.98.0.42 -t 301fc382-cf06-4a1b-b5ef-0c8c45273aef
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md:625]:
```suggestion
   ./installer.sh  -c install -u https://10.98.0.42 -t 301fc382-cf06-4a1b-b5ef-0c8c45273aef
```

File: docs/content/stable/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md [hunk 3]
```diff
@@ -700,13 +700,13 @@ To reconfigure a node for use in a different provider, do the following:
...
     ```sh
-    node-agent node configure -t <api_token> -u https://<yba_address>:9000
+    node-agent node configure -t <api_token> -u https://<yba_address>
     ```
 
     For example, if you run the following:
 
     ```sh
-    node-agent node configure -t 1ba391bc-b522-4c18-813e-71a0e76b060a -u http://10.98.0.42:9000
+    node-agent node configure -t 1ba391bc-b522-4c18-813e-71a0e76b060a -u http://10.98.0.42
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md:709]:
```suggestion
    node-agent node configure -t 1ba391bc-b522-4c18-813e-71a0e76b060a -u https://10.98.0.42
```

File: docs/content/v2.18/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md [hunk 4]
```diff
@@ -726,13 +726,13 @@ To install the YugabyteDB node agent manually, as the `yugabyte` user, do the fo
...
    ```sh
-   ./installer.sh -c install -u https://<yba_address>:9000 -t <api_token>
+   ./installer.sh -c install -u https://<yba_address> -t <api_token>
    ```
 
    For example, if you run the following:
 
    ```sh
-   ./installer.sh  -c install -u http://10.98.0.42:9000 -t 301fc382-cf06-4a1b-b5ef-0c8c45273aef
+   ./installer.sh  -c install -u http://10.98.0.42 -t 301fc382-cf06-4a1b-b5ef-0c8c45273aef
```
Comment (aishwarya24) [inline docs/content/v2.18/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md:735]:
```suggestion
   ./installer.sh  -c install -u https://10.98.0.42 -t 301fc382-cf06-4a1b-b5ef-0c8c45273aef
```

**Review by aishwarya24 — APPROVED** [review]
LGTM. @nkhogen - If my suggestions are right and the rest looks correct to you, I can commit them and merge this PR for @ddhodge.

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md (hunk 1 above)
Comment (nkhogen) [reply docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md:707]: +1

**Review by nkhogen — APPROVED** [review]

**Pull request closed without merging**